src/components/authentication/signup.html

```diff
@@ -23,40 +23,40 @@
         <!-- HEADER NAVIGATION -->
        <nav class="nav-container">
            <div class="brand">
-               <img src="../../images/logo.png" alt="muffin-logo" class="logo">
+               <img src="../../images/logo.webp" alt="muffin-logo" class="logo">
                <a href="../../../index.html" class="brand-name">Bakin Lane</a>
                <a href="../../../index.html" class="nav-item">Home</a>
                <a href="../product/productlist.html" class="nav-item">Buy Now</a>
            </div> 
            <ul>
                <div class="searchbar-container">
-                    <button class="nav-search-button">
+                    <button class="nav-search-button" aria-label="search">
                         <i class="fa fa-solid fa-search"></i>
                     </button>
                     <input type="text" placeholder="Type to search" class="nav-search-field">
```

Comment:
here you can use ```type="search"``` also

Reply:
Thanks for pointing it out, have made the changes